src/components/list/PlaceItem.tsx

```diff
@@ -8,18 +8,25 @@ import { Place } from '@/services/list/types';
 
 interface PlaceItemProps {
   data: Place;
+  station: string;
 }
 
 function PlaceItem(props: PlaceItemProps) {
   const {
     data: { id, place, count, hashtags = [], distance, imageURL },
+    station,
   } = props;
 
   return (
     <Link
-      href={`/detail/${id}`}
       key={id}
       className="mb-4 flex w-full border-b-2 border-GRAY_100 pb-4"
+      href={{
+        pathname: `/detail/${id}`,
+        query: {
+          station: station,
+        },
+      }}
```

Comment:
공간 상세 뷰 API에서 사용되는 station 값을 넘겨줍니다!

Reply:
객체의 key, value값이 같으면 `station` 으로만 써줘도 될 것 같아요!
```suggestion
      href={{
        pathname: `/detail/${id}`,
        query: {
          station,
        },
      }}
```